BinaryObjectScanner/Protection/LaserLok.cs

```diff
@@ -149,6 +150,41 @@ public List<string> CheckDirectoryPath(string path, List<string>? files)
 
             return MatchUtil.GetFirstMatch(path, matchers, any: true);
         }
+        
+        public string? CheckISO(string file, ISO9660 iso, bool includeDebug)
+        {
+            #region Initial Checks
+            
+            if (iso.VolumeDescriptorSet[0] is not PrimaryVolumeDescriptor pvd)
+                return null;
+            
+            
+            if (FileType.ISO9660.NoteworthyApplicationUse(pvd))
+                return null; //TODO: this might be too unsafe until more App Use strings are known
+            
+            if (!FileType.ISO9660.NoteworthyReserved653Bytes(pvd))
+                return null;
+            
+            #endregion
+            
+            var reserved653Bytes = pvd.Reserved653Bytes;
+            var firstNonZero = Array.FindIndex(reserved653Bytes, b => b != 0);
```

Comment:
I think this returns `-1` if the index can't be found. That will probably throw an exception when trying to read the null-terminated string.